run-tests.sh

```diff
@@ -42,6 +42,16 @@ check_pytest () {
     python setup.py test
 }
 
+if [ $# -eq 0 ]; then
+    check_script
```

Comment:
You can also amend the names so that they would appear the same, i.e. generally make them lowercase.

E.g. in REANA we have `lint-black`, here we have `Lint-black`...